wdl/tasks/Sniffles2.wdl

```diff
@@ -0,0 +1,171 @@
+version 1.0
+
+
+import "Structs.wdl"
+
+
+workflow Sniffles2 {
+
+    meta {
+        description: "This workflow calls SV candidates using Sniffles2 population mode."
+    }
+
+    input {
+        Array[File] sampleBAMs
+        Array[File] sampleBai
+        Int minsvlen
+        String prefix
+        String sample_id
```

Comment:
This should be an array because it's a multi-sample workflow.
Sorry to have missed this.